ai_economist/real_business_cycle/README.md

```diff
@@ -0,0 +1,120 @@
+# Real Business Cycle (RBC)
+This directory implements a **Real-Business-Cycle** (RBC) simulation with many heterogeneous, interacting strategic agents of various types, such as **consumers, firms, and the government**. For details, please refer to this paper `ADD PAPER LINK HERE`. We also provide training code that uses deep multi-agent reinforcement learning to determine optimal economic policies and dynamics in these many agent environments. Below are instructions required to launch the training runs.
```

Comment:
ADD PAPER LINK HERE -> "Finding General Equilibria in Many-Agent Economic Simulations using Deep Reinforcement Learning (ArXiv link forthcoming)"